CHANGES.rst

```diff
@@ -1,6 +1,41 @@
 Release Notes
 =============
 
+1.4.5
+-----
+
+This release adds support for validating pex entrypoints at build time in addition to several bugfixes.
+
+* Fix PEX environment setup. (#531)
+  `#531 <https://github.com/pantsbuild/pants/pull/531>`_
```

Comment:
noting that these urls all seem to reference `pantsbuild/pants` vs `pantsbuild/pex` in URI string - and so crossref old pants issues on click.

Reply:
Facepalm. Thanks for spotting that. Now fixed on master but I don't think there is a way to fix on PyPI.

Reply:
nbd - should correct itself going forward on the next release.